pom.xml

```diff
@@ -221,6 +227,19 @@
 			<version>1.9.5</version>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-simple</artifactId>
+			<version>1.7.21</version>
+		</dependency>
+
+		<dependency>
+			<groupId>com.github.tomakehurst</groupId>
+			<artifactId>wiremock-jre8-standalone</artifactId>
+			<version>2.32.0</version>
+			<scope>test</scope>
+		</dependency>
```

Comment:
I guess this used to mock the HTTP users? This library comes from GeoTools.

Reply:
It is used to test mock endpoints